### PR TITLE
json-glib 1.0.4

### PR DESCRIPTION
**Pull request by tschoonj**
version bump
brew audit --strict compliance

### CODE REVIEW

File: Library/Formula/json-glib.rb [hunk 1]
```diff
@@ -21,6 +19,18 @@ def install
                           "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
                           "--enable-introspection=yes"
-    system "make install"
+    system "make", "install"
+  end
```
Comment (xu-cheng) [inline Library/Formula/json-glib.rb]: Please leave a line between the install and test.

File: Library/Formula/json-glib.rb [hunk 2]
```diff
@@ -21,6 +19,18 @@ def install
...
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+      #include <json-glib/json-glib.h>
+
+      int main(int argc, char *argv[]) {
+        JsonParser *parser = json_parser_new();
+        return 0;
+      }
+    EOS
+    system ENV.cc, "-I#{HOMEBREW_PREFIX}/include/json-glib-1.0", "-I#{HOMEBREW_PREFIX}/include/glib-2.0", "-I#{HOMEBREW_PREFIX}/lib/glib-2.0/include", "-I#{HOMEBREW_PREFIX}/opt/gettext/include", "test.c", "-L#{HOMEBREW_PREFIX}/lib", "-L#{HOMEBREW_PREFIX}/lib", "-L#{HOMEBREW_PREFIX}/opt/gettext/lib", "-ljson-glib-1.0", "-lgio-2.0", "-lgobject-2.0", "-lglib-2.0", "-lintl", "-o", "test"
```
Comment (xu-cheng) [inline Library/Formula/json-glib.rb]:
```ruby
glib = Formula["glib"]
system ENV.cc, "test.c", "-I#{include}", "-I#{glib.opt_include}", "-L#{lib}", "-L#{glib.opt_lib}", "-l...", "-o", "test"
```

You don't need add `"-L#{HOMEBREW_PREFIX}/lib", "-L#{HOMEBREW_PREFIX}/opt/gettext/lib"`.

**Comment by tschoonj** [comment]
I added an empty line before the `test do` block. You may want to add that to the `audit --strict` tests.

**Comment by tschoonj** [comment]
Mountain Lion build failed for some reason... could you trigger it again? Thanks!

**Comment by tschoonj** [comment]
@xu-cheng any idea how to fix this? `ENV["CPPFLAGS"]` and/or `ENV["LDFLAGS"]` returns nil causing split to fail. This works fine on my local machine though. Is it even necessary to use these environment variables? I never did so in the past and things worked without a hitch

**Comment by xu-cheng** [comment]
I have updated the tool again to address this.

**Comment by tschoonj** [comment]
ok thanks, json-glib and librsvg have been queued for rebuilding
Since libxml++ didn't suffer from this problem, can you close the pull request?

**Comment by xu-cheng** [comment]
Thanks @tschoonj

**Pull request closed without merging**